### DISCUSSION TITLE
Search Results web part Refinement Filters syntax

### DISCUSSION THREAD

**Discussion by jmuehlenbein** [microsoft-search/pnp-modern-search#2060] [Q&A] [answered]
Hiya, In the results web part, I'm trying to enter two different refinement filters: SPS-HideFromAddressLists:0 and RefinableString00:equals("foo"). I can get either of them to work alone but not sure what the syntax would be to include both?

**Comment by FranckyC** [comment, marked as answer]
Hi @jmuehlenbein, 

Refinement filters use FQL and the syntax should be `and(SPS-HideFromAddressLists:0,RefinableString00:equals("foo"))`. See https://docs.microsoft.com/en-us/sharepoint/dev/general-development/fast-query-language-fql-syntax-reference#and

Anyway refinement filters is more a legacy feature and you could achieve the same using a KQL query in the query template like:

`RefinableString00:foo SPS-HideFromAddressLists:0`

**Reply by jmuehlenbein** [reply]
Thanks for the clarifications, @FranckyC! Much appreciated!
